Client/multiplayer_sa/CMultiplayerSA_WidescreenFix.cpp

```diff
@@ -0,0 +1,1111 @@
+    fMiscWidth[20] = 0.0015625f * fWideScreenWidthScale * fHudWidthScale;
+    fMiscWidth[21] = 0.0015625f * fWideScreenWidthScale * fHudWidthScale;
+    fMiscWidth[22] = 0.0015625f * fWideScreenWidthScale * fHudWidthScale;
+    fMiscWidth[23] = 0.0015625f * fWideScreenWidthScale * fHudWidthScale;
+    fMiscWidth[24] = 0.0015625f * fWideScreenWidthScale * fHudWidthScale;
+    fMiscWidth[25] = 0.0015625f * fWideScreenWidthScale * fHudWidthScale;
+    fMiscWidth[26] = 0.0015625f * fWideScreenWidthScale * fHudWidthScale;
+    fMiscWidth[27] = 0.0015625f * fWideScreenWidthScale * fHudWidthScale;
+    fMiscWidth[28] = 0.0015625f * fWideScreenWidthScale;
+
+    fMiscHeight[0] = 0.002232143f * fWideScreenHeightScale;            // StretchY
+    fMiscHeight[1] = 0.078125f * fWideScreenHeightScale;
+    fMiscHeight[2] = 0.66964287f * fWideScreenHeightScale;
+    fMiscHeight[3] = 0.13392857f * fWideScreenHeightScale;
+    fMiscHeight[4] = 0.6473214f * fWideScreenHeightScale;
+    fMiscHeight[5] = 0.22321428f * fWideScreenHeightScale;
+    fMiscHeight[6] = 0.24553572f * fWideScreenHeightScale;
+    fMiscHeight[7] = 0.24553572f * fWideScreenHeightScale;
+    fMiscHeight[8] = 0.22321428f * fWideScreenHeightScale;
+    fMiscHeight[9] = 0.77678573f * fWideScreenHeightScale;
+    fMiscHeight[10] = 0.75446427f * fWideScreenHeightScale;
+    fMiscHeight[11] = 0.75446427f * fWideScreenHeightScale;
+    fMiscHeight[12] = 0.77678573f * fWideScreenHeightScale;
+    fMiscHeight[13] = 0.037946429f * fWideScreenHeightScale;
+    fMiscHeight[14] = 0.002232143f * fWideScreenHeightScale;
+    fMiscHeight[15] = 0.002232143f * fWideScreenHeightScale * fHudHeightScale;
+    fMiscHeight[16] = 0.002232143f * fWideScreenHeightScale * fHudHeightScale;
+    fMiscHeight[17] = 0.002232143f * fWideScreenHeightScale * fHudHeightScale;
+    fMiscHeight[18] = 0.002232143f * fWideScreenHeightScale * fHudHeightScale;
+    fMiscHeight[19] = 0.002232143f * fWideScreenHeightScale * fHudHeightScale;
+    fMiscHeight[20] = 0.002232143f * fWideScreenHeightScale * fHudHeightScale;
+    fMiscHeight[21] = 0.002232143f * fWideScreenHeightScale * fHudHeightScale;
+    fMiscHeight[22] = 0.002232143f * fWideScreenHeightScale * fHudHeightScale;
+    fMiscHeight[23] = 0.002232143f * fWideScreenHeightScale * fHudHeightScale;
+    fMiscHeight[24] = 0.002232143f * fWideScreenHeightScale * fHudHeightScale;
+    fMiscHeight[25] = 0.002232143f * fWideScreenHeightScale * fHudHeightScale;
+    fMiscHeight[26] = 0.002232143f * fWideScreenHeightScale * fHudHeightScale;
+    fMiscHeight[27] = 0.002232143f * fWideScreenHeightScale * fHudHeightScale;
+    fMiscHeight[28] = 0.002232143f * fWideScreenHeightScale * fHudHeightScale;
+}
+
+void UpdateScriptFixes()
+{
+    float w;
+    if (*pfScreenAspectRatio < fDefaultWidth)
+        w = static_cast<float>(pRsGlobal->width);
+    else
+        w = pRsGlobal->width * fDefaultWidth / *pfScreenAspectRatio;
```

Comment:
You can use the conditional operator (? :) here for better readability and make "w" as constant

Reply:
reviewing the code I realize this function is not used